src/utils/parser/getSymbols.js

```diff
@@ -1,11 +1,13 @@
-import { getAst } from "./utils/ast";
-import traverse from "babel-traverse";
+// @flow
+
+import { traverseAst } from "./utils/ast";
 import { isVariable, isFunction } from "./utils/helpers";
-import isEmpty from "lodash/isEmpty";
 import * as t from "babel-types";
 
 import getFunctionName from "./utils/getFunctionName";
 
+import type { SourceText } from "debugger-html";
+
 const symbolDeclarations = new Map();
 
 export type ASTLocation = {
```

Comment:
this feels a bit of out of place here -- maybe it should also be moved to the utils/ast? I might also not be understanding the code